bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -16,21 +16,31 @@
     (url('foundation.index'), 'index', 'The Mozilla Foundation'),
     (url('foundation.about'), 'about', 'About the Foundation'),
     ('https://mzl.la/foundation-strategy', 'strategy', '2016-2018 Strategy'),
-    (url('foundation.annualreport'), 'report', 'Annual Report'),
-    ('https://careers.mozilla.org/', 'careers', 'Careers'),
-    (donate_url('foundation_nav'), 'donate', 'Donate'),
     (url('foundation.licensing'), 'licensing', 'Licensing & Trademarks'),
     (url('foundation.documents.index'), 'documents', 'Public Documents'),
+    ('https://careers.mozilla.org/', 'careers', 'Careers'),
 ] %}
 
-{% block content %}
-  <main id="main-content">
-    {% block article_title %}{% endblock %}
-
-    <div class="main-column">
-      {% block article %}{% endblock %}
-    </div>
+{% set programs = [
+  (url('foundation.issues'), 'issues', 'Key Internet Issues'),
+  (url('foundation.advocacy'), 'advocacy', 'Advocacy'),
+  (url('foundation.leadership-network'), 'leadership-network', 'Leadership Network')
+] %}
 
+{% block content %}
+  <main>
+    <header class="page-header">
+      <h1><span class="content">{{ _('The Mozilla Foundation') }}</span></h1>
+    </header>
+    <section>
+      <div class="content content-main">
+      <div class="main-column">
+        <header class="section-header">
+        <h1>{% block article_title %}{% endblock %}</h1>
```

Comment:
First paragraph: Huh, TIL. Didn't realize that was abandoned.

Second paragraph: I'm not sure I understand what you're suggesting- I wrapped the `article_title` and `article_subtitle` in tags so that there'd be markup to refer back and forth when creating new pages in the future/making changes to styles.

Are you suggesting something in each of the page templates like this:
```
{% block article_head %}
<h1> Article title </h1>
<p> Article subtitle </p>
{% endblock %}
```

And then CSS like this?
```
h1 + p { padding-bottom: 1em; background: /*thin stripe image*/ }
```